apps/filecoin-site/src/app/[locale]/provide-storage/page.tsx

```diff
@@ -39,54 +31,73 @@ export default function ProvideStorage() {
         structuredData={generateStructuredData(PROVIDE_STORAGE_SEO)}
       />
 
-      <GradientContainer className="lg:pb-20">
-        <Navigation backgroundVariant="transparentDark" />
+      <Navigation backgroundVariant="dark" />
+      <div className="relative isolate">
```

Comment:
I think the image should merge into the nav? Like currently on Store Data and Learn? Instead of being cut off like this? 

<img width="4128" height="606" alt="Image" src="https://github.com/user-attachments/assets/ddeb6002-047b-4dc1-b32c-100f08dac75a" />

Reply:
And actually like the current Provide Storage page as well..

Reply:
@barbaraperic Looking great to me! Let's Get This Merged! ❤️